frontend/src/pages/projects/screens/projects/ProjectListView.tsx

```diff
@@ -42,73 +41,87 @@ const ProjectListView: React.FC<ProjectListViewProps> = ({
-    showPagination && (
-      <Pagination
-        dropDirection={pageDirection}
-        perPageComponent="button"
-        itemCount={filteredProjects.length}
-        perPage={pageSize}
-        page={page}
-        onSetPage={(e, newPage) => setPage(newPage)}
-        onPerPageSelect={(e, newSize, newPage) => {
-          setPageSize(newSize);
-          setPage(newPage);
-        }}
-        widgetId="table-pagination"
-      />
-    );
-
   const searchTypes = React.useMemo(() => Object.keys(SearchType), []);
 
+  const [deleteData, setDeleteData] = React.useState<ProjectKind | undefined>();
+  const [editData, setEditData] = React.useState<ProjectKind | undefined>();
+  const [refreshIds, setRefreshIds] = React.useState<string[]>([]);
+
   return (
-    <>
-      <Toolbar>
-        <ToolbarContent>
-          <ToolbarItem>
-            <SearchField
-              types={searchTypes}
-              searchType={searchType}
-              searchValue={search}
-              onSearchTypeChange={(searchType) => {
-                setSearchType(searchType);
-              }}
-              onSearchValueChange={(searchValue) => {
-                setPage(1);
-                setSearch(searchValue);
-              }}
-            />
-          </ToolbarItem>
-          <ToolbarItem>
-            <NewProjectButton />
-          </ToolbarItem>
-          <ToolbarItem>
-            <Button variant="link">
-              <Link to="/notebookController">Launch Jupyter</Link>
+    <React.Fragment>
+      <Table
+        enablePagination
+        minPageSize={1}
+        data={filteredProjects}
+        columns={columns}
+        emptyTableView={
+          <div style={{ textAlign: 'center' }}>
+            No projects match your filters.{' '}
+            <Button variant="link" isInline onClick={resetFilters}>
+              Clear filters
             </Button>
-          </ToolbarItem>
-          <ToolbarItem variant="pagination" alignment={{ default: 'alignRight' }}>
-            {pagination('down')}
-          </ToolbarItem>
-        </ToolbarContent>
-      </Toolbar>
-      <ProjectTable
-        clearFilters={resetFilters}
-        projects={filteredProjects.slice(pageSize * (page - 1), pageSize * page)}
-        getColumnSort={sort.getColumnSort}
-        refreshProjects={refreshProjects}
-      />
-      {showPagination && (
-        <Toolbar>
-          <ToolbarContent>
-            <ToolbarItem variant="pagination" alignment={{ default: 'alignRight' }}>
-              {pagination('up')}
+          </div>
+        }
+        Row={(project) => (
+          <ProjectTableRow
+            key={project.metadata.uid}
+            obj={project}
+            isRefreshing={refreshIds.includes(project.metadata.uid || '')}
+            setEditData={(data) => setEditData(data)}
+            setDeleteData={(data) => setDeleteData(data)}
+          />
+        )}
+        toolbarContent={
+          <React.Fragment>
```

Comment:
```suggestion
          <>
```

Reply:
any reason against react.fragment?

Reply:
Just looks odd to me -- only reason to ever use it is when you need to `key` it as well. I suppose it's okay -- no real issue -- I've done React for a long time and we always linted this out -- so unless we do, I suppose it does not matter.